cogs/greetings.py

```diff
@@ -2,7 +2,7 @@
 from discord.ext.commands.context import Context
 
 from .base import CommandBase
-
+import random
```

Comment:
We want consistent formatting across everyone. I made a script that does this for us on the readme

run:
`python fmt.py`

this will format the code and sort the imports